docs/analyzer-rules/AZFW0014.md

```diff
@@ -0,0 +1,19 @@
+# AZFW0011: Missing Registration for ASP.NET Core Integration
+
+| | Value |
+|-|-|
+| **Rule ID** |AZFW00014|
+| **Category** |[Usage]|
+| **Severity** |Warning|
+
+## Cause
+
+This rule is triggered when the ASP.NET function app is missing registration for the appropriate method.
+
+## Rule description
+
+The ASP.NET function app should have registration for `ConfigureFunctionsWebApplication` method. This rule will be violated when the app is missing this registration.
+
+## How to fix violations
```

Comment:
Can we add a `When to suppress violations` section, saying it is okay to suppress this if `ConfigureFunctionsWebApplication` is called indirectly through a different method off the `IHostBuilder`.

Reply:
done